zulip/integrations/bridge_between_zulips/run-interrealm-bridge

```diff
@@ -83,10 +89,16 @@ if __name__ == "__main__":
     # A bidirectional tunnel
     pipe_event1 = create_pipe_event(client2, bot1, bot2, args.stream)
     p1 = mp.Process(target=client1.call_on_each_event, args=(pipe_event1, ["message"]))
-    pipe_event2 = create_pipe_event(client1, bot2, bot1, args.stream)
-    p2 = mp.Process(target=client2.call_on_each_event, args=(pipe_event2, ["message"]))
-    p1.start()
-    p2.start()
-    print("Listening...")
-    p1.join()
-    p2.join()
+
+    if args.oneway:
+        p1.start()
+        print("Listening...")
+        p1.join()
+    else:
+        pipe_event2 = create_pipe_event(client1, bot2, bot1, args.stream)
+        p2 = mp.Process(target=client2.call_on_each_event, args=(pipe_event2, ["message"]))
+        p1.start()
+        p2.start()
+        print("Listening...")
+        p1.join()
+        p2.join()
```

Comment:
Can we do this without duplicating code between the two paths?

Reply:
This is the simplest way to do it without having to add more lines.

Reply:
I'm not sure if this is cleaner to read, but if this is equivalent then you could have:
```python
    1 if not args.oneway:
    2     pipe_event2 = ...
    3     p2 = mp.Process(...)
    4     p2.start()
    5 p1.start()
    6 print("Listening")
    7 p1.join()
    8 if not args.oneway
    9     p2.join()
```
Does the `start()` order matter?

Reply:
Yeah, it is possible to do it that way too.